project/build.properties

```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-sbt.version=1.9.3
+sbt.version=1.10.11
```

Comment:
Testing is being conducted based on https://github.com/apache/spark/pull/50999. Any unrelated changes will be rebased after https://github.com/apache/spark/pull/50999 is merged.